pretty_printers_tests/tests/cell.rs

```diff
@@ -0,0 +1,44 @@
+// === LLDB TESTS ==================================================================================
+
+// lldb-command:run
+
+// lldb-command:print cell
+// lldbr-check:[...]cell = { value = 42 }
+// lldbg-check:[...]$0 = { value = 42 }
+// lldb-command:print ref_cell1
+// lldbr-check:[...]ref_cell1 = borrow=1 { value = 42 }
+// lldbg-check:[...]$1 = borrow=1 { value = 42 }
+// lldb-command:print ref1
+// lldbr-check:[...]ref1 = borrow=1 { [...] = 42 }
+// lldbg-check:[...]$2 = borrow=1 { [...] = 42 }
+// lldb-command:print ref_mut2
+// lldbr-check:[...]ref_mut2 = borrow_mut=1 { [...] = 42 }
+// lldbg-check:[...]$3 = borrow_mut=1 { [...] = 42 }
+
+// === GDB TESTS ==================================================================================
+
+// gdb-command:run
+
+// gdb-command:print cell
+// gdb-check:[...]$1 = {value = 42}
+// gdb-command:print ref_cell1
+// gdb-check:[...]$2 = borrow=1 = {value = 42, borrow = 1}
+// gdb-command:print ref1
+// gdb-check:[...]$3 = borrow=1 = {[...] = 42, borrow = 1}
+// gdb-command:print ref_mut2
+// gdb-check:[...]$4 = borrow_mut=1 = {[...] = 42, borrow = -1}
```

Comment:
Do I understand correctly that we don't provide `borrow` field in lldb pretty-printer because it can show summary but gdb one cannot?

Reply:
Yes